docs/reference/msbuild-targets.md

```diff
@@ -336,9 +336,14 @@ Additional restore settings may come from MSBuild properties in the project file
 | RestoreConfigFile | Path to a `Nuget.Config` file to apply. |
 | RestoreNoCache | If true, avoids using cached packages. See [Managing the global packages and cache folders](../consume-packages/managing-the-global-packages-and-cache-folders.md). |
 | RestoreIgnoreFailedSources | If true, ignores failing or missing package sources. |
+| RestoreFallbackFolders | Fallback folders, used in the same way the user packages folder is used. |
+| RestoreAdditionalProjectSources | Additional sources to the resulting list of sources |
+| RestoreAdditionalProjectFallbackFolders | Additional fallback folders appended to the resulting list of fallback folders |
+| RestoreAdditionalProjectFallbackFoldersExcludes | Excludes fallback folders specified in `RestoreAdditionalProjectFallbackFolders` |
 | RestoreTaskAssemblyFile | Path to `NuGet.Build.Tasks.dll`. |
 | RestoreGraphProjectInput | Semicolon-delimited list of projects to restore, which should contain absolute paths. |
-| RestoreOutputPath | Output folder, defaulting to the `obj` folder. |
+| RestoreUseSkipNonexistentTargets  | When the projects are collected via MSBuild it determines whether they collected using the `SkipNonexistentTargets` optimization. When not set, defaults to `true`. Consequence is fail-fast behavior when a project's targets cannot be imported. |
```

Comment:
whether they **are** collected.

**The** consequence is **a** fail-fast behaviour